catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -131,14 +132,11 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
               targetRowHeight={220}
             />
           </GalleryContainer>
-        </ul>
+        </PlainList>
       )}
 
       {(!isFullSupportBrowser || isSmallGallery) && (
-        <ul
-          className="flex flex--wrap plain-list no-padding no-margin"
-          role="list"
-        >
+        <PlainList className="flex flex--wrap" role="list">
```

Comment:
This one could be made into another component (`styled(PlainList)`) in the aim of removing `flex` classes later too!

Reply:
Oh neat, I didn't know you could do that. I'll do that now!